File: Backend/App.Domain/Entities/Worker.cs
namespace App.Domain.Entities
{
    public class Worker
    {
        public Guid WorkerId { get; set; }
        public string Name { get; set; } = null!;
        public int Age { get; set; }
        public string Email { get; set; } = null!;
        public string Phone { get; set; } = null!;
        public string? Picture { get; set; }
        public Guid OrgId { get; set; }
        public Org Org { get; set; } = null!;
        public ICollection<FarmWorker> FarmWorkers { get; set; } = new List<FarmWorker>();

    }
}

File: frontend/src/types/worker.ts

export interface WorkerResponse {
        workerId: string;
        name:string,
        age: number;
        email: string;
        picture: string;
        phone: string;
}

export type Roles = "CEO" | "Captain" | "Worker";

export interface AssignWorkerFormResponse {
        farmId: string;
        workerId: string;
        role: Roles;
        certifiedUntil: string;
}

export interface FarmWorkerDetails {
        workerId: string
        workerName: string
        workerEmail: string
        role: Roles;
        certifiedUntil: string | null
}

export interface FarmWDetailsDto{
        farmId: string;
        farmName: string;
        role: Roles;
        certifiedUntil: string | null;
}
